docs/configuration-reference.md

```diff
@@ -536,7 +536,6 @@ You can split the config into smaller modules by using the value `!include file.
 
 ```yaml
 skills: !include skills.yaml
-
 ```
 
-_Note: The file.yaml that you wish to include in the config must be in the same directory as your configuration.yaml (e.g ~/.opsdroid)_
+Since we use [appdirs](https://pypi.org/project/appdirs/) to keep our opsdroid files in a logical location depending on what operative system you use, the included file should be located on your [user configuration directory](https://github.com/ActiveState/appdirs#some-example-output).
```

Comment:
I don't think the users need to know that we use `appdirs` for this. It's probably unnecessary technical noise. Maybe just saying that files are stored in appropriate user configuration directories depending on which operating system you use.

Reply:
Good point will fix this 👍